src/server/func_services/func_server.js

```
@@ -274,16 +289,20 @@ async function invoke_func(req) {
         aws_config: _make_aws_config(req),
         rpc_options: _make_rpc_options(req),
     };
-    if (node) {
-        dbg.log0('invoking on node',
-            func.name, req.params.event,
-            node.name, node.pool);
-        res = await server_rpc.client.func_node.invoke_func(params, {
-            address: node.rpc_address
-        });
-    } else {
-        dbg.log0('invoking on server', func.name, req.params.event);
-        res = await server_func_node.invoke_func({ params });
+    try {
+        if (node) {
+            dbg.log0('invoking on node',
+                func.name, req.params.event,
+                node.name, node.pool);
+            res = await server_rpc.client.func_node.invoke_func(params, {
+                address: node.rpc_address
+            });
+        } else {
+            dbg.log0('invoking on server', func.name, req.params.event);
+            res = await server_func_node.invoke_func({ params });
+        }
+    } catch (e) {
+        dbg.error('invoke returned with error:', e);
```

Add a throw here to not eat the errors.